.circleci/config.yml

```diff
@@ -1329,6 +1419,86 @@ jobs:
           mentions: <<parameters.slack-mentions>>
           channel: <<parameters.slack-channel>>
 
+  test-e2e-gke-cuda-11:
+    parameters:
+      cluster-id-prefix:
+        type: string
+      mark:
+        type: string
+      parallelism:
+        type: integer
+        default: 1
+      gke-version:
+        type: string
+        default: "1.18.12-gke.1205"
+      machine-type:
+        type: string
+        default: "a2-highgpu-1g"
+      num-machines:
+        type: integer
+        default: 1
+      gpu-type:
+        type: string
+        default: "nvidia-tesla-a100"
+      gpus-per-machine:
+        type: integer
+        default: 1
+      region:
+        type: string
+        default: "us-central1"
+      node-locations:
+        type: string
+        default: "us-central1-c"
+      slack-mentions:
+        type: string
+        default: ""
+      slack-channel:
+        type: string
+        default: ""
+    docker:
+      - image: determinedai/cimg-base:stable
+    parallelism: <<parameters.parallelism>>
+    environment:
+      # Using the TF2-only image here is a hack to get more test coverage since TF1 is
+      # still the default system-wide. TF1 tests that fail in this configuration should
+      # be skipped when the "CUDA" environment variable is set to 11
+      TF1_CPU_IMAGE: determinedai/environments:cuda-11.0-pytorch-1.7-tf-2.4-gpu-0.9.0
```

Comment:
Why do we have to set the cpu images to gpu?  That seems like it should be unnecessary.

Reply:
Good point... I thought a lot about making the TF1_GPU image to the cuda-11 image, but I guess for CPU it adds no additional value that we're not already testing.